links.html

```diff
@@ -44,13 +37,20 @@
     <h1> Useful Links </h1>
         <ul>
             <li> <a href="https://validator.w3.org/nu/" target="_blank">HTML Validator </a> </li> 
+            <li> <a href="https://jigsaw.w3.org/css-validator/" target = "_blank">CSS Validator</a>  </li>
+            <li> <a href="https://developer.mozilla.org/en-US/" target = "_blank">Devloper Tools</a>  </li>
+            <li> <a href="https://caniuse.com/" target = "_blank">CanIuse</a>  </li>
+            <li> <a href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" target = "_blank">Normalized CSS</a>  </li>
+            <li> <a href="https://fontpair.co/" target = "_blank">Font Pair</a>  </li>
+            <li> <a href="https://fonts.google.com/" target = "_blank">Google Fonts</a>  </li>
+            <li><a href="https://realfavicongenerator.net/" target="_blank">Favicon Generator</a></li>
         </ul>
```

Comment:
Note the use of "_blank" instead of "blank" for the target.  The difference is important